session/database.rst

```diff
@@ -0,0 +1,547 @@
+.. index::
+    single: Session; Database Storage
+
+Store Sessions in a Database
+============================
+
+Symfony stores sessions in files by default. If your application is served by
+multiple servers, you'll need to use instead a database to make sessions work
+across different servers.
+
+Symfony can store sessions in all kinds of databases (relational, NoSQL and
+key-value) but recommends key-value databases like Redis to get best performance.
+
+Store Sessions in a key-value Database (Redis)
+----------------------------------------------
+
+This section assumes that you have a fully-working Redis server and have also
+installed and configured the `phpredis extension`_.
+
+First, define a Symfony service for the connection to the Redis server:
+
+.. code-block:: yaml
+
+    # config/services.yaml
+    services:
+        # ...
+        Redis:
+            class: Redis
+            calls:
+                - method: connect
+                  arguments:
+                      - '%env(REDIS_HOST)%'
+                      - '%env(int:REDIS_PORT)%'
+
+                # uncomment the following if your Redis server requires a password
+                # - method: auth
+                #   arguments:
+                #       - '%env(REDIS_PASSWORD)%'
+
+                # uncomment the following if your Redis server defines a key prefix
+                # - method: setOption
+                #   arguments:
+                #       - !php/const Redis::OPT_PREFIX
+                #       - 'my_prefix'
+
+Now pass this Redis connection as an argument of the service associated to the
+:class:`Symfony\Component\HttpFoundation\Session\Storage\Handler\RedisSessionHandler`:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # config/services.yaml
+        services:
+            # ...
+            Symfony\Component\HttpFoundation\Session\Storage\Handler\RedisSessionHandler:
+                arguments:
+                    - '@Redis'
```

Comment:
Note, this can also be some other types, like `\RedisCluster` or [Predis](https://github.com/nrk/predis):
https://github.com/symfony/symfony/blob/21a05de9bf196fe98d567c7bf0da14ce63922866/src/Symfony/Component/HttpFoundation/Session/Storage/Handler/RedisSessionHandler.php#L51

Not sure what's the policy of documenting those, but you probably have it for Cache component somewhere (since it uses exactly the same drivers) so could probably link to it or repeat it here?